exes/mqtt-broadcaster/inc/mqtt_broadcaster.hpp

```diff
@@ -84,11 +96,51 @@ class mqtt_broadcaster {
 
 private:
   auto initialize() -> asio::awaitable<void> {
+    create_scada_signals();
     co_await connect_to_broker();
     add_new_signals();
     co_await ncmd_listener();
   }
 
+  auto create_scada_signals() -> void {
+    auto config_scada_signals = config_.value().scada_signals;
+
+    std::vector<std::string> allowed_types = { "_bool", "_double_t", "_int64_t", "_json", "_string", "_uint64_t" };
```

Comment:
this will become unnecessary if you use `type_e` as the config parameter, as mentioned in config file